client/my-sites/checkout/composite-checkout/types/transaction-endpoint.ts

```diff
@@ -75,6 +77,18 @@ export type WPCOMTransactionEndpointDomainDetails = {
 	postalCode: string;
 };
```

Comment:
Aren't we missing properties such as `organization`, `address_2` and `alternateEmail`? I think `address_1` and `address_2` should be written `address1` and `address2`. That's the format used by the old checkout according to p99Zz8-1in-p2#comment-3770.